com.unity.netcode.gameobjects/Runtime/Timing/IRealTimeProvider.cs

```diff
@@ -0,0 +1,10 @@
+namespace Unity.Netcode
+{
+    public interface IRealTimeProvider
```

Comment:
I'm also wondering if we could mock time for tests specifically without adding/exposing anything new to public.

Reply:
In the real world, NGO wouldn't mock time, it'd just use UnityEngine runtime driven `Time.XXX` internally everywhere.

Reply:
you're right, this should be internal, not public